azure-eventhubs-eph/src/main/java/com/microsoft/azure/eventprocessorhost/PartitionManager.java

```diff
@@ -27,11 +31,10 @@
 	// Protected instead of private for testability
     protected final EventProcessorHost host;
     protected Pump pump;
-
-    private String partitionIds[] = null;
+    protected String partitionIds[] = null;
```

Comment:
>String [](start = 14, length = 6)

use 'volatile' for fields which are accessed by multiple threads and doesn't use thread synchronization...

Reply:
Done